docs(public_api): change description

lightningtgc opened the pull request:
Benchpress has been moved to angular/angular in modules/@angular/benchpress

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)



**What is the new behavior?**



**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...


**Other information**:

lightningtgc committed:
doc(public_api): change description

Benchpress has been moved to angular/angular in modules/@angular/benchpress

chuckjaz commented on docs/PUBLIC_API.md:
```diff
@@ -19,7 +19,7 @@ One intentional omission from this list is `@angular/compiler`, which is current
 Additionally only the command line usage (not direct use of APIs) of `@angular/compiler-cli` is covered.
 
-Other projects developed by the Angular team like angular-cli, Angular Material, benchpress, will be covered by these or similar guarantees in the future as they mature.
+Other projects developed by the Angular team like angular-cli, Angular Material, Universal, will be covered by these or similar guarantees in the future as they mature.
```
`Universal` is not part of `@angular/angular` and should not b added. `benchpress` should be kept.

vicb commented:
Please also reword your comment as `docs(): ...` instead of `doc()`. Thanks

lightningtgc commented:
@vicb thx

vicb commented:
Imeant the commit comment, not the GH issue headline, please update that as well as edits suggested by @chuckjaz. Thanks.

lightningtgc committed:
docs(public_api): change description

Here means 'other projects',like angular-cli, Angular Material. And as we know, benchpress project has been moved to angular/angular in modules/@angular/benchpress. It should not be 'other projects'.
